rules/ml/ml_linux_system_network_connection_discovery.toml

```
@@ -0,0 +1,45 @@
+[metadata]
+creation_date = "2020/09/03"
+ecs_version = ["1.6.0"]
+maturity = "development"
```

```suggestion
maturity = "production"
```

If you want it in the package